src/resources/views/crud/inc/datatables_logic.blade.php

```diff
@@ -50,7 +49,14 @@
 
     @endif
         if (saved_list_url && persistentUrl!=window.location.href) {
+
+            //we save the current alerts into local storage so we can read it later in alerts.blade.php
+            //after the user had been redirected by persistent table
+            localStorage.setItem('backpack_alerts', JSON.stringify(@json(\Alert::getMessages())));
+
+            //finally redirect the user.
```

Comment:
In the future @pxpm please put a space between `//` and the actual comment. So:
```suggestion
            // finally redirect the user.
```

So far it hasn't been a priority to do these kind of improvements, but now that there's 3 of us and I know @promatik  cares about this kind of stuff too, so let's take a closer look at how our comments look & read too, so it looks like they've been written by a team not individuals, and don't distract from the code itself. Plus, excellent comments are a sign of excellent code. 

Side-story: maybe you've noticed that in Laravel itself, code blocks always form a triangle downwards. So:

```php
    /**
     * Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis non mi nec 
     * orci euismod venenatis. Integer quis sapien et diam facilisis ultricies 
     * quis justo. Phasellus sem turpis, ornare quis aliquet ut, volutpat 
     * et lectus aliquam a egestas elit.
     */
    
    // NOT
    
    /**
     * Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis non mi nec orci
     * euismod venenatis. Integer quis sapien et diam facilisis facilisis ultricies quis 
     * justo. Phasellus sem turpis, ornare quis aliquet ut, volutpat et lectus. Aliquam 
     * a egestas elit.
     */
```

This is super SUPER minor. But it's something I felt the need to let you guys know - let's start paying attention at these kind of things too, in the code we write from now on. 

Of course, it'd be great if we had a tool that did that for us, like it does for PHP, but... yeah... I'll investigate that too, maybe there is one we can use. Pretty difficult to find one that knows what programming/scripting language it is, automatically (PHP/JS/CSS) inside blade files...